--- libtheora-1.1.1/configure.ac	2014-06-04 23:27:25.000000000 +0100
+++ libtheora-1.1.1/configure.ac	2014-06-04 23:29:04.000000000 +0100
@@ -168,7 +168,7 @@
         case $host in 
         *)
                 DEBUG="-g -Wall -Wno-parentheses -DDEBUG -D__NO_MATH_INLINES"
-                CFLAGS="-Wall -Wno-parentheses -O3 -fforce-addr -fomit-frame-pointer -finline-functions -funroll-loops"
+                CFLAGS="-Wall -Wno-parentheses -O3 -fomit-frame-pointer -finline-functions -funroll-loops"
                 PROFILE="-Wall -Wno-parentheses -pg -g -O3 -fno-inline-functions -DDEBUG";;
         esac
 fi
